dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/uploadconfig/functional/FrameworkBooleanLambda.kt

```diff
@@ -0,0 +1,15 @@
+package org.dataland.frameworktoolbox.specific.uploadconfig.functional
+
+/**
+ * A TypeScript lambda of the format (dataset: FrameworkDataType): boolean => BODY
+ */
+class FrameworkBooleanLambda(lambdaBody: String, imports: Set<String> = emptySet()) : FrameworkLambda(
```

Comment:
Can someone briefly explain to me this FrameworkLambda type? Is this something like a generic type?

Reply:
Gladly. A `FrameworkLambda` is a typescript lambda function that has one input: The framework (i.e., functions of the form `(frameworkDataType: FrameworkTypeHere) => { /* FUNCTION */}`

Reply:
thank you